.github/ISSUE_TEMPLATE/bug-performance-report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug/Performance report
+about: Create a report to help us improve
+title: "[Bug/Performance]"
+labels:
+assignees: ''
+
+---
+
+**Describe the bug**
+
+A clear and concise description of what the bug is.
+
+**System information**
+- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
+- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
+- HW Platform (e.g. iPhone 8, Pixel 2, Samsung Galaxy, Intel based Mac, i7 based Dell with NVIDIA GPGPU etc., or Google Cloud machine)
```

Comment:
Is this really needed? At the current projects scope I don't think we'll be running addons on on embedded devices etc. If we do run into that though it'll be an edge case not a template issue. We have the OS info, so maybe just a Yes/No GPU question?

Reply:
> just a Yes/No GPU question

+1 for simplicity

Reply:
Okay done